enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -225,6 +228,11 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
       }
     } catch (ExpressionEvaluationException ex) {
       throw new EnforcerRuleException("Unable to lookup an expression " + ex.getMessage(), ex);
+    } finally {
+      // Always show artifact problems. But these should not be the cause of failing the rule.
+      for (ArtifactProblem problem : artifactProblems) {
+        logger.warn(problem.toString());
```

Comment:
This is warn because having a unresolvable Maven artifact should not cause the build failure as long as there is no linkage errors.

Reply:
make this a comment in the code